web/src/services/analytics/events.ts

```diff
@@ -109,4 +109,9 @@ export type Events = {
         elementId?: string
         meta?: apiObject
     }
+    video_played: {
+        videoId: string
+        referenceId?: string
+        data?: Record<string, unknown>
+    }
```

Comment:
Can you also update https://github.com/bcc-code/bcc-media-adr/blob/master/02-analytics/technical.md

Reply:
https://github.com/bcc-code/bcc-media-adr/pull/30